### PR TITLE
Fix `oc set env` example

### PR DESCRIPTION
**Pull request by ammachado**
Example was showing a command line where it executes `oc set env` without parameters, while the correct is `env` only.

### CODE REVIEW

File: pkg/cli/set/env.go [hunk 1]
```diff
@@ -78,7 +78,7 @@ var (
 		oc set env -f dc.json ENV-
 
 		# Set some of the local shell environment into a deployment config on the server
-		oc set env | grep RAILS_ | oc env -e - dc/myapp
+		env | grep RAILS_ | oc env -e - dc/myapp
```
Comment (soltysh) [inline pkg/cli/set/env.go:81]: Can we instead just drop this example, it doesn't gives us too much value but only clutters the overall help page for this command?

**Review by soltysh — CHANGES_REQUESTED** [review]
/ok-to-test

File: pkg/cli/set/env.go (hunk 1 above)
Comment (soltysh) [reply pkg/cli/set/env.go:81]: @ammachado I'm open to get that also removed from `kubectl` as well.

Comment (ammachado) [reply pkg/cli/set/env.go]: Example removed.

**Pull request closed without merging**